### PR TITLE
Add Alibaba Druid plugin

### PR DESCRIPTION
**Pull request by VictorZeng**
add alibaba druid plugin

- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Related #7735.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
Pending on https://github.com/apache/skywalking-java/pull/26 review.

File: oap-server/server-starter/src/main/resources/component-libraries.yml [hunk 1]
```diff
@@ -374,6 +374,9 @@ Sentinel:
 GuavaCache:
   id: 114
   languages: Java
+druid:
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/component-libraries.yml:377]:
```suggestion
AlibabaDruid:
```

According to https://github.com/apache/skywalking-java/pull/27/files#r711735928

**Comment by wu-sheng** [comment]
Wait for https://github.com/apache/skywalking-java/pull/27 ready to merge.

**Comment by wu-sheng** [comment]
> Update the CHANGES log.

Notice, you clicked this, but you don't update the changes.md.

File: CHANGES.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ Release Notes.
 * Improve the speed of writing TiDB by batching the SQL execution.
 * Fix wrong service name when IP is node IP in `k8s-mesh`.
 * Support dynamic configurations for openAPI endpoint name grouping rule.
+* Add component definition for `Alibaba Druid`.
```
Comment (wu-sheng) [inline CHANGES.md:72]: FYI @VictorZeng This is the changelog for main repo. From SkyWalking backend perspective, we don't care about the plugin added at agent side.

**Review by wu-sheng — APPROVED** [review]

File: CHANGES.md (hunk 2 above)
Comment (VictorZeng) [reply CHANGES.md:72]: OK, I get it now.

**Pull request merged**